### PR TITLE
Added LAN client and logging to LAN Server

### PR DESCRIPTION
**Pull request by PrabhanshuAttri**
#102

### CODE REVIEW

File: hardware/CommunicationsPi/lan_client.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+logging.info('Pinging')
+while True:
+    try:
+        payload = {
+            'key1': 'value1',
+            'key2': 'value2'
+        }
+        logging.info('data: ' + json.dumps(payload))
+        response = requests.post(url, data=payload)
+        response.raise_for_status()
```
Comment (ab7289) [inline hardware/CommunicationsPi/lan_client.py]: is the lan_client only for sending data 1 way or will it be checking for responses as well?

Comment (PrabhanshuAttri) [reply hardware/CommunicationsPi/lan_client.py]: We would be checking the response status code and response as well just to make sure that we don't discard sensor data before it is logged into the system. If any exception (line 22-25) occurs, we'll have to add handlers in such situations.

Comment (ab7289) [reply hardware/CommunicationsPi/lan_client.py]: Ah gotcha sounds good. I will approve this PR, would you mind creating a story to capture adding the exception handlers?

Comment (PrabhanshuAttri) [reply hardware/CommunicationsPi/lan_client.py]: Great call. I think it should be a part of the local data store. WIll create a ticket for that.

**Pull request merged**